src/orquestra/sdk/_base/cli/_dorq/_entry.py

```diff
@@ -171,6 +171,27 @@ def status():
 )
 
 
+@dorq.command()
+@cloup.option(
+    "-s",
+    "--server",
+    required=True,
```

Comment:
**suggestion**: please describe the `-s` and `-t` options, ideally with example values. `help=` kwarg can be used for this